docs/editor/integrated-terminal.md

```diff
@@ -176,7 +176,7 @@ To remove entries from the terminal dropdown, set the name of the profile to `nu
 
 By default, the task/debug features will use the default profile. To override that, use the `terminal.integrated.automationShell.<platform>` setting:
 
-```jsonc
+```json
```

Comment:
`jsonc` is the language type id for JSON with comments, which this code block is using.